docker-compose.yml

```diff
@@ -59,7 +59,7 @@ services:
       - SSAS_PUBLIC_URL=${SSAS_PUBLIC_URL}
       - SSAS_TIMEOUT_MS=2000
       - BCDA_ENABLE_NEW_GROUP=true
-      - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993,A9994
+      - PRIORITY_ACO_REG_EX=([A-Z]99([0-8]|9[0-8])$$) # $$ signifies a literal $ in the string
```

Comment:
Should we also in include `^` at the beginning? And also include the possibility of [A-Z]999/9999

Reply:
The carrot shouldn't affect anything UNLESS we ever think we are going to get bad id names that start with garbage before the initial [A-Z]. We DID talk about the quadruple 9 though and I need to update that.

Reply:
Updated and included a picture from the regex to show what it validates currently.